tests/ui/tests/support/input.ts

```diff
@@ -1,8 +1,10 @@
 Cypress.Commands.add('inputClearAndType', (selector: string, newValue: string): void => {
     cy.get(selector)
         .find('input')
+        .as('inpt')
         .scrollIntoView()
-        .click({force: true})
-        .clear({force: true})
-        .type(newValue, {force: true});
+        .click({ force: true })
+    cy.get('@inpt')
```

Comment:
Why do we use an alias? Did we fix some issue with this? Maybe it's worth a comment to remember it in the future.

Reply:
There was a problem when you click for this use case with the warning and displaying extern type, page was refreshing and it was breaking saying chain of commands fail because DOM changed and it can not be executed further, suggesting to break the chain with this alias.